### PR TITLE
replace sun.awt.OSInfo and add JAXB jar

### PR DESCRIPTION
**Pull request by ianturton**
Replace calls to OSInfo with commons-lang3 equivaltents and added a dependency to JAXB api as it is no longer shipped with JDK-11

### CODE REVIEW

File: modules/plugin/coverage-multidim/coverage-api/pom.xml [hunk 1]
```diff
@@ -93,6 +93,15 @@
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+    </dependency>
+    <dependency>
```
Comment (aaime) [inline modules/plugin/coverage-multidim/coverage-api/pom.xml]: Why is JAXB needed here?

Comment (ianturton) [reply modules/plugin/coverage-multidim/coverage-api/pom.xml]: Not sure, but it fails to build on my machine without it (as at least a test dependency):

```
Running org.geotools.imageio.GeoSpatialImageReaderTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.003 s <<< FAILURE! - in org.geotools.imageio.GeoSpatialImageReaderTest
[ERROR] testReader(org.geotools.imageio.GeoSpatialImageReaderTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalArgumentException: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at org.geotools.imageio.GeoSpatialImageReaderTest.testReader(GeoSpatialImageReaderTest.java:102)
Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at org.geotools.imageio.GeoSpatialImageReaderTest.testReader(GeoSpatialImageReaderTest.java:102)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
	at org.geotools.imageio.GeoSpatialImageReaderTest.testReader(GeoSpatialImageReaderTest.java:102)
```

Comment (ianturton) [reply modules/plugin/coverage-multidim/coverage-api/pom.xml]: there is another commit coming (maybe) that centralizes the version and makes it a test dependency

Comment (aaime) [reply modules/plugin/coverage-multidim/coverage-api/pom.xml]: So the jaxb dependency should be coming from imagemosaic, which is a dependency of coverage-api. But repeating it does not hurt.

**Comment by aaime** [comment]
Looks good to me

**Pull request merged**